prow/config/config_test.go

```diff
@@ -26,3 +28,26 @@ func TestConfigLoads(t *testing.T) {
 		t.Fatalf("Could not load config: %v", err)
 	}
 }
+
+func TestConfigSecurityJobsMatch(t *testing.T) {
+	c, err := Load("../config.yaml")
+	if err != nil {
+		t.Fatalf("Could not load config: %v", err)
+	}
+	kp := c.Presubmits["kubernetes/kubernetes"]
+	sp := c.Presubmits["kubernetes-security/kubernetes"]
+	for i, j := range kp {
+		name := strings.Replace(j.Name, "pull-kubernetes", "pull-security-kubernetes", -1)
+		if name != sp[i].Name {
+			t.Fatalf("%s should match %s", name, sp[i].Name)
```

Comment:
Make this error more helpful. Someone is going to add a presubmit to k/k without knowing about k-s/k. When that happens, the error message should exactly tell them what is wrong and how to fix it.